edxval/api.py

```diff
@@ -652,22 +652,36 @@ def copy_course_videos(source_course_id, destination_course_id):
             )
 
 
-def export_to_xml(edx_video_id, course_id=None):
+def export_to_xml(video_ids, course_id=None, external=False):
     """
-    Exports data about the given edx_video_id into the given xml object.
+    Exports data for a video into an xml object.
 
-    Args:
-        edx_video_id (str): The ID of the video to export
+    NOTE: For external video ids, only transcripts information will be added into xml.
```

Comment:
Additional note: If external=True, then `edx_video_id` is going to be on first index of the list.